library/include/hipblas.h

```diff
@@ -18426,6 +18543,58 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasGemmEx(hipblasHandle_t    handle,
 
     - Supported types are determined by the backend. See rocBLAS/cuBLAS documentation.
 
+    With HIPBLAS_V2 define, hipblasGemmBatchedEx accepts hipDataType for aType, bType, and cType.
+    It also accepts hipblasComputeType_t for computeType. hipblasGemmBatchedEx will no
+    longer support hipblasDataType_t for these parameters in a future release.
+
+        #ifdef HIPBLAS_V2 // available in hipBLAS version 2.0.0 and later with -DHIPBLAS_V2
+
+            hipblasStatus_t hipblasGemmBatchedEx(hipblasHandle_t      handle,
+                                                 hipblasOperation_t   transA,
+                                                 hipblasOperation_t   transB,
+                                                 int                  m,
+                                                 int                  n,
+                                                 int                  k,
+                                                 const void*          alpha,
+                                                 const void*          A[],
+                                                 hipDataType          aType,
+                                                 int                  lda,
+                                                 const void*          B[],
+                                                 hipDataType          bType,
+                                                 int                  ldb,
+                                                 const void*          beta,
+                                                 void*                C[],
+                                                 hipDataType          cType,
+                                                 int                  ldc,
+                                                 int                  batchCount,
+                                                 hipblasComputeType_t computeType,
+                                                 hipblasGemmAlgo_t    algo)
+
+        #else // Deprecated
+
```

Comment:
to be consistent can this be changed from `Deprecated` to `[DEPRECATED]`

Reply:
Updated in next commit, along with renaming the sample executable to hipblas-example-gemmEx_v2.